### PR TITLE
Machine learning service

### PR DESCRIPTION
**Pull request by DilanOjeda**
File upload and download functionalities have been added to the machine learning service.

### CODE REVIEW

File: MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    });
+    // TODO: calls the ObjectRecognition class
+
+    res.json({
+        msg: 'Images were uploaded.',
+        images: imagesploaded
+    });
+}
+
+// downloads image
```
Comment (stospina) [inline MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js]: Omit the comment

File: MACHINE-LEARNING-SERVICE/src/routes/objectRecognition.routes.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const {Router} = require('express');
+
+const {recognizeObjects, downloadImage} = require('../controllers/objectRecognition.controller');  
+const {uploadImagesMiddleware} = require('../middlewares/multer.middleware');
+const router = Router();
+
+// Routes uris
```
Comment (stospina) [inline MACHINE-LEARNING-SERVICE/src/routes/objectRecognition.routes.js]: omit the comment

**Review by stospina — CHANGES_REQUESTED** [review]

File: MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js (hunk 1 above)
Comment (DilanOjeda) [reply MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js]: Please check again. I've already removed the comments.

**Review by stospina — APPROVED** [review]

File: MACHINE-LEARNING-SERVICE/src/routes/objectRecognition.routes.js (hunk 2 above)
Comment (DilanOjeda) [reply MACHINE-LEARNING-SERVICE/src/routes/objectRecognition.routes.js]: Please check again. I've already removed the comments.

**Review by stospina — APPROVED** [review]
It's OK

**Review by luisdavidparra — COMMENTED** [review]
Looks good

**Pull request merged**